extensions/amp-taboola/0.1/amp-taboola.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+AMP.registerElementAlias("amp-taboola", "amp-ad", {type: 'taboola'});
```

Comment:
I don't think we want to do this. This creates an additional HTTP request with zero benefit instead of the slightly nicer markup.

What is the reason not just to use `amp-ad` with type `taboola`?